### PR TITLE
[connect > portal] - add warning message for all the transfer of USDC that does not use CCTP

### PR DESCRIPTION
**Pull request by yuli-ferna**
The condition follows these priority of the checks:

- Check whether the asset is registered on the dest chain
- Check whether the user has an associated token account created for that asset
- Check whether the chain pair supports CCTP (USDC only)

Result if the feature it's on:
image

Issue: https://github.com/wormhole-foundation/wormhole-connect/issues/1233

### CODE REVIEW

File: wormhole-connect/src/config/index.ts [hunk 1]
```diff
@@ -49,6 +49,9 @@ export const ATTEST_URL =
     ? ''
     : 'https://wormhole-foundation.github.io/example-token-bridge-ui/#/register';
 
+export const PORTAL_USDC_BRIDGE_URL =
+  'https://www.portalbridge.com/usdc-bridge/';
```
Comment (sebastianscatularo) [inline wormhole-connect/src/config/index.ts]: could we make it configurable? and if not, could we remove the `www.`

**Review by sebastianscatularo — CHANGES_REQUESTED** [review]

File: wormhole-connect/src/views/Bridge/Inputs/TokenWarnings.tsx [hunk 2]
```diff
@@ -205,10 +215,24 @@ function TokenWarnings() {
   ]);
 
   useEffect(() => {
+    // check if the token is USDC and the chains involved are not CCTP
+    const usdcAndNoCCTP =
```
Comment (sebastianscatularo) [inline wormhole-connect/src/views/Bridge/Inputs/TokenWarnings.tsx:221]: also, we may want this to be configurable to allow integrators to enable/disable at their convenience, not sure if all wants to get redirect to portal bridge

Comment (yuli-ferna) [reply wormhole-connect/src/views/Bridge/Inputs/TokenWarnings.tsx:221]: Done!!

File: wormhole-connect/src/config/index.ts (hunk 1 above)
Comment (yuli-ferna) [reply wormhole-connect/src/config/index.ts]: Done!!

File: wormhole-connect/src/views/Bridge/Inputs/TokenWarnings.tsx [hunk 3]
```diff
@@ -7,7 +7,7 @@ import {
   setAssociatedTokenAddress,
   setForeignAsset,
 } from 'store/transferInput';
-import { ATTEST_URL, TOKENS } from 'config';
+import { ATTEST_URL, PORTAL_USDC_BRIDGE_URL, TOKENS } from 'config';
```
Comment (sebastianscatularo) [inline wormhole-connect/src/views/Bridge/Inputs/TokenWarnings.tsx]: maybe not PORTAL, just bridge :)

**Review by sebastianscatularo — DISMISSED** [review]

File: wormhole-connect/src/views/Bridge/Inputs/TokenWarnings.tsx (hunk 3 above)
Comment (yuli-ferna) [reply wormhole-connect/src/views/Bridge/Inputs/TokenWarnings.tsx]: done!

**Review by sebastianscatularo — DISMISSED** [review]

**Pull request closed without merging**